pubsub/cloud-client/quickstart.py

```diff
@@ -19,76 +19,110 @@
 
 
 def end_to_end(project_id, topic_name, subscription_name, num_messages):
-    # [START pubsub_end_to_end]
+    # [START pubsub_quickstart_end2end_basic]
     import time
 
+    from google.api_core.exceptions import NotFound
     from google.cloud import pubsub_v1
 
     # TODO project_id = "Your Google Cloud Project ID"
```

Comment:
Isn't there a way to grab this from the environment, in case it's configured. 
So something like
```
# replace with you own way of grabbing project id
project_id = google_magic_client_library_that_understands_env_variables.get_active_project()
```

Reply:
Yes there is! Maybe we can address this in a different PR because currently we use the same TODO format for all our code samples. What do you think?

```Python
project_id = os.environ['GCLOUD_PROJECT']
```

Reply:
sg.  not critical.